app/controllers/fed_ax_api_controller.rb

```diff
@@ -0,0 +1,128 @@
+require "#{ Rails.root }/lib/shipping_api.rb"
+
+class FedAxApiController < ApplicationController
+  include FedAxApiWrapper
+
+  def quote
```

Comment:
This is a very long method. You might want to consider extracting pieces out into smaller methods.